modules/acs-cloud-requirements.adoc

```diff
@@ -43,7 +43,8 @@ Memory:: 6 GiB of RAM is required.
 See the default memory and CPU requirements for each component and ensure that the node size can support them.
 ====
 
-Storage:: For {product-title-managed-short}, a persistent volume claim (PVC) is not required. However, a PVC is strongly recommended if you have secured clusters with Scanner V4 enabled. Use Solid-State Drives (SSDs) for best performance. However, you can use another storage type if you do not have SSDs available.
+Storage:: For {product-title-managed-short}, a persistent volume claim (PVC) is not required. However, a PVC is strongly recommended if you have secured clusters with Scanner V4 enabled.
+Use Solid-State Drives (SSDs) for best performance. However, you can use another storage type if you do not have SSDs available.
```

Comment:
Is the only change here the carriage return before _Use Solid-State Drives_? Might make the PR a bit easier to remove the carriage return.